src/app/layout.tsx

```diff
@@ -18,11 +18,14 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className + ` layout py-3`}>
+      <body className={inter.className + ` layout py-3 `}>
         <Header />
         {children}
-        <Footer />
       </body>
+      <Footer />
```

Comment:
Footer should  come within body tag or else it may cause hydration errors 
just make sure that this order is maintained 
```
<body>
<Header />
{children}
<Footer />
</body>
```

its fine even if the footer does not cover the 100vw of the screen